pkg/operator/ceph/object/health.go

```diff
@@ -42,26 +42,26 @@ type bucketChecker struct {
 	context         *clusterd.Context
 	objContext      *Context
 	interval        time.Duration
-	serviceIP       string
-	port            string
+	port            int32
 	client          client.Client
 	namespacedName  types.NamespacedName
 	healthCheckSpec *cephv1.BucketHealthCheckSpec
 	isExternal      bool
+	sslDisabled     bool
```

Comment:
It's a minor thing, but I think the logic will be easier to maintain with enabled instead of disabled in the name. 
```suggestion
	sslEnabled     bool
```